envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -74,6 +74,7 @@ class TlsAuthenticationProperties {
 }
 
 class TlsProtocolProperties {
+    var cipherSuites: Set<String> = setOf("ECDHE-ECDSA-AES128-GCM-SHA256", "ECDHE-RSA-AES128-GCM-SHA256")
```

Comment:
It should be a list, or a Set implementation that preserves order, because order matters for cipher suites